src/main/kotlin/parts/wisdom/arcraider/visualization/SwingApp.kt

```diff
@@ -1,22 +1,66 @@
 package parts.wisdom.arcraider.visualization
 
+import parts.wisdom.arcraider.Task
 import parts.wisdom.arcraider.loadTaskFromFile
+import java.awt.Dimension
 import java.awt.EventQueue
+import java.awt.FlowLayout
 import java.io.File
+import javax.swing.*
 
 private const val WINDOW_HEIGHT_PIXEL = 600
 private const val WINDOW_WIDTH_PIXEL = 800
 
+private var currentGridPanel : GridPane? = null
```

Comment:
I don't think this value is ever used as null. We should make it a `lateinit var` so it can be non nullalble.

Reply:
fixed